MatrixSDK/JSONModels/Media/MXURLPreview.m

```diff
@@ -0,0 +1,121 @@
+// 
+// Copyright 2021 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "MXURLPreview.h"
+
+@implementation MXURLPreview
+
++ (instancetype)modelFromJSON:(NSDictionary *)JSONDictionary
+{
+    MXURLPreview *urlPreview = [MXURLPreview new];
+    
+    NSString *siteName, *title, *description, *imageURL, *imageType;
+    NSNumber *imageWidth, *imageHeight, *imageFileSize;
+    
+    MXJSONModelSetString(siteName, JSONDictionary[@"og:site_name"]);
```

Comment:
Do not hesitate to create constants when you use a key twice. Like `static NSString * const kSiteNameJSONKey = @"og:site_name";`

Reply:
Good point, that's much better. Are these good, kept private to the implementation?